src/components/AvatarGroup/AvatarGroupCounterTooltipContentVirtualizedList.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Ref, useMemo } from "react";
+import VirtualizedList from "../VirtualizedList/VirtualizedList";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { AvatarTypes } from "../Avatar/AvatarConstants";
+import { AvatarProps } from "../Avatar/Avatar";
+import styles from "./AvatarGroupCounterTooltipContentVirtualizedList.module.scss";
+
+const LIST_OPTIONS = Object.freeze({
+  maxItemsWithoutScroll: 10,
+  itemLineHeight: 34,
+  itemLineWidth: 150
+});
+
+interface AvatarGroupCounterTooltipContentVirtualizedListProps extends VibeComponentProps {
+  /**
+   * Array of Avatar components
+   */
+  avatarItems?: { value: AvatarProps & { tooltipContent: any } }[];
```

Comment:
tooltip content can always be string or react element no?

Reply:
Replaced with `tooltipContent: string | ReactElement`